src/test/java/hw3/ex2/HomeAndElementsCompositeTest.java

```diff
@@ -0,0 +1,53 @@
+package hw3.ex2;
+
+import hw3.AbstractTestBase;
+import hw3.ex1.HomePage;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomeAndElementsCompositeTest extends AbstractTestBase {
+
+    @Test
+    public void logInGoToElementsTest() {
+        ElementsPage elementsPage = new ElementsPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        driver.get(credentialsHelper.getProperty("url"));
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //logInAndCheckUserName
+        homePage.getHeaderMenuComposite().openLogInForm();
+        homePage.getHeaderMenuComposite().enterUserName(credentialsHelper.getProperty("login"));
+        homePage.getHeaderMenuComposite().enterPassword(credentialsHelper.getProperty("password"));
+        homePage.getHeaderMenuComposite().clickLoginBtn();
+
+        assertEquals(homePage.getHeaderMenuComposite().getLoggedInUserName(), "ROMAN IOVLEV");
+
+        //getToElementsPage
+        homePage.getHeaderMenuComposite().clickOnTheServiceTab();
+        homePage.getHeaderMenuComposite().clickOnTheDifferentElements();
+
+        //Select checkboxes Water, Wind
+        elementsPage.getElementMetalColorComposite().clickWaterCheckBox();
+        elementsPage.getElementMetalColorComposite().clickWindCheckBox();
+
+        //Select radio selen
+        elementsPage.getElementMetalColorComposite().clickSelenRadio();
+
+        //Select in dropdown Yellow
+        elementsPage.getElementMetalColorComposite().clickColorsDropDown();
+        elementsPage.getElementMetalColorComposite().clickYellow();
```

Comment:
It will be better send names of the checkbox, radiobutton and select as parameter to the page method